dockstore-webservice/src/main/java/io/dockstore/webservice/core/Image.java

```diff
@@ -89,12 +97,14 @@ public Image() {
 
     }
 
-    public Image(List<Checksum> checksums, String repository, String tag, String imageID, Registry imageRegistry) {
+    public Image(List<Checksum> checksums, String repository, String tag, String imageID, Registry imageRegistry, Long size, String imageUpdateDate) {
```

Comment:
Why is it a string and not the standard epoch time? Is it due to the API we are invoking? If it is due to the API, I suppose you're constrained, but curious if it is it an ISO-8601 format?

Reply:
GitLab, Docker Hub, and Quay.io returns different times. The TRS time format is also not specified